tools/deploy_prow.sh

```diff
@@ -37,6 +37,10 @@ function launchConfig(){
   kubectl create secret generic hmac-token --from-literal=hmac="$(./tools/1password.sh -d config/prow/hmac-token)" || true
   kubectl create secret generic oauth-token --from-literal=oauth="$(./tools/1password.sh -d config/prow/oauth-token)" || true
   kubectl create secret generic oauth-token --from-literal=oauth="$(./tools/1password.sh -d config/prow/oauth-token)" -n test-pods || true
+
+  # PR Status
+  kubectl create secret generic github-oauth-config --from-literal=oauth="$(./tools/1password.sh -d prow-prstatus-github-oauth-app.yaml)" || true
```

Comment:
Would it make sense to add this to the root [Makefile](https://github.com/falcosecurity/test-infra/blob/master/Makefile) too? WDYT?

Reply:
Oh sure, that `Makefile` is to be updated with this

Reply:
Also, I think that we should unify scripts with `Makefile`s without repeating